src/Polly.Core/Retry/RetryHelper.cs

```diff
@@ -114,7 +114,7 @@ private static TimeSpan DecorrelatedJitterBackoffV2(int attempt, TimeSpan baseDe
 
     private static TimeSpan ApplyJitter(TimeSpan delay, Func<double> randomizer)
     {
-        var offset = (delay.TotalMilliseconds * JitterFactor) / 2;
+        var offset = delay.TotalMilliseconds * JitterFactor / 2;
```

Comment:
Put this one back as it was please - it's less mental gymnastics to remember the precedence rules.